packages/manager/modules/billing-components/src/components/commitment-duration/commitment-duration.controller.js

```diff
@@ -7,6 +7,11 @@ export default class {
     this.$attrs = $attrs;
     this.BillingService = BillingService;
     this.coreConfig = coreConfig;
+
+    this.COMMITMENT_DURATION = {
+      DEFAULT: 'default',
+      P12M: 'P12M',
```

Comment:
Nitpic: for better readability, consider using proper constant name like
```suggestion
      TWELVE_MONTHS: 'P12M',
```